use-shopping-cart/src/index.js

```diff
@@ -144,7 +146,7 @@ export const CartProvider = ({
   successUrl,
   cancelUrl,
   currency,
-  language = navigator.language,
+  language = isClient ? navigator.language : 'en-US',
```

Comment:
I think this should just be `language` as `formatCurrencyString` will handle what happens if it's `undefined`. It'll still be optional we just don't need to handle the default value here anymore.